Source/Core/AudioCommon/AOSoundStream.h

```diff
@@ -19,6 +20,7 @@ class AOSound final : public SoundStream
 {
 #if defined(HAVE_AO) && HAVE_AO
 	std::thread thread;
+	std::atomic<bool> m_run_thread;
```

Comment:
I checked Flag's source and all it is, is just an atomic under the hood that does the exact same thing, so I'll keep things as is, since all it requires in that case are the standard library headers.